util/formula_templater/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/hashicorp/homebrew-tap/util/formula_templater
+
+go 1.16
+
+require github.com/hashicorp/hcl v1.0.0
```

Comment:
FYI: It shouldn't affect the syntax you use above, but you might want to use v2 here, v1 is not actively maintained anymore.